src/test/java/com/google/api/codegen/testdata/discoveries/js/js_billingtest.v1.json.baseline

```diff
@@ -0,0 +1,361 @@
+
+<html>
```

Comment:
Is there a way to avoid having baselines for the languages we aren't supporting for disco gapic yet? Maybe we need a new directory for discogapic to cut down on noise.